sql/core/src/test/resources/sql-tests/inputs/collations.sql

```diff
@@ -101,3 +101,319 @@ select str_to_map(text collate utf8_binary, pairDelim collate utf8_lcase, keyVal
 select str_to_map(text collate utf8_binary, pairDelim collate utf8_binary, keyValueDelim collate utf8_binary) from t4;
 
 drop table t4;
+
+-- ConcatWs
+create table t1(utf8_binary string collate utf8_binary, utf8_lcase string collate utf8_lcase) using parquet;
```

Comment:
we shouldn't create a new table for every single expression

let's try in this PR to consolidate as many expressions as possible to use a single table (or a small / fixed / constant number of tables) - think of ways to keep it simple, yet offer complete testing coverage